src/components/ShellBox/__tests__/shell-box.test.tsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import React from 'react';
+import renderer from 'react-test-renderer';
+import ShellBox from '../index';
+
+describe('ShellBox component', () => {
```

Comment:
Add the return type of void
See the other test files for examples

Reply:
added in f257538